VikingQuiz.UI/src/Components/StartGame/StartGame.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import './StartGame.css';
+import HomeButton from '../Buttons/HomeButton/HomeButton';
+import BottomLogo from '../BottomLogo/BottomLogo';
+
+class StartGame extends React.Component<any, any> {
+    constructor(props: any) {
+      super(props);
+    }
+
+
+
+    public render() {
+        const randomstring = require("randomstring");
+        const code = randomstring.generate({
+            length: 6,
+            charset: 'abcdefghijklmnopqrstuvwxyz0123456789'
+        });
+        return (
+            <div className="codeForm">
+                <div className="container">
+                    <div className="row">
+                        <div className="col-xs-10 col-xs-offset-1 col-md-6 col-md-offset-3">
+                            <HomeButton/>
+                        </div>
+                    </div>
+                    <div className="row">
+                        <div className="col-xs-10 col-xs-offset-1 col-md-6 col-md-offset-3">
+                            <div className="codeLabel"> YOUR CODE </div>
```

Comment:
It would be cool if we can extract this string "YOUR CODE" to some constant, so that we can easy change it in future (or maybe in case we want to add localization).